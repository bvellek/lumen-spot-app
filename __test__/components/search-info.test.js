import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import SearchInfo from '../../public/js/components/search-info';

describe('<SearchInfo />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <SearchInfo />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
