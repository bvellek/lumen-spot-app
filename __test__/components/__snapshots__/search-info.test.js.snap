// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchInfo /> it renders correctly 1`] = `
<div>
  <Connect(SearchFormContainer) />
  <Connect(SearchCoordContainer) />
  <hr />
</div>
`;
